### ISSUE TITLE
How to Validate Phone Number

### ISSUE THREAD

**Issue by indumathymk** [yeojz/otplib#607]
Is there is an option to validate given phone number is Mobile or Landline?

**Comment by sayerrrr** [comment]
No, you need to use a service like Twilio verify or something similiar.

**Comment by reachbharathan** [comment]
This cannot be verified by this repo, maybe it need to be validated via paid service.

**Comment by yeojz** [comment]
As mentioned by other replies, this library handles only OTP generation using TOTP/HOTP. 
Mobile numbers are out of scope.

If you're referring to sending OTPs via mobile or telephony services, that's out of scope for this library.
Thanks.
